extensions/gmsa-dc/v1/GenerateCredentialSpecResource.ps1

```diff
@@ -0,0 +1,66 @@
+<#
```

Comment:
General comment on this file: could be better to port the changes you've made to https://github.com/kubernetes-sigs/windows-gmsa/blob/master/scripts/GenerateCredentialSpecResource.ps1 and then just pull it from there; will help keep everything in sync and less work to maintain it all, what do you think?

Reply:
I have made a PR against the original script and changed the referencing script to pull from the repo location you have suggested above.

https://github.com/kubernetes-sigs/windows-gmsa/pull/16